demos/calibration/calibration-pubmed-link-prediction.ipynb

```diff
@@ -66,33 +68,35 @@
     "### Loading the PubMed Diabetes network data"
    ]
   },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "**Downloading the dataset:**\n",
-    "    \n",
-    "The dataset used in this demo can be downloaded from https://linqs-data.soe.ucsc.edu/public/Pubmed-Diabetes.tgz\n",
-    "\n",
-    "The following is the description of the dataset:\n",
-    "\n",
-    "> The Pubmed Diabetes dataset consists of 19717 scientific publications from PubMed database \n",
-    "> pertaining to diabetes classified into one of three classes. The citation network consists \n",
-    "> of 44338 links. Each publication in the dataset is described by a TF/IDF weighted word \n",
-    "> vector from a dictionary which consists of 500 unique words.\n",
-    "\n",
-    "Download and unzip the `Pubmed-Diabetes.tgz` file to a location on your computer.\n",
-    "\n",
-    "Set the `data_dir` variable to point to the location of the processed dataset."
-   ]
-  },
   {
    "cell_type": "code",
    "execution_count": 2,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "The PubMed Diabetes dataset consists of 19717 scientific publications from PubMed database pertaining to diabetes classified into one of three classes. The citation network consists of 44338 links. Each publication in the dataset is described by a TF/IDF weighted word vector from a dictionary which consists of 500 unique words."
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "PubMed Diabetes dataset is already downloaded\n"
```

Comment:
It's probably annoying and unnecessary, but this looks like it was the old output before we switched to using `logging`. Is it worth rerunning the notebooks to switch back to the new output?

Reply:
notebooks have been reformatted and re-run (via format_notebooks with some changes in #733).